vuepress/docs/docs/adapters.md

```diff
@@ -55,6 +55,12 @@ That's it! `http4sRoute` is a valid http4s route ready to serve our API.
 ::: tip Limitations
 The zio-http interpreter in tapir does not include multipart and websocket support.
 
-Caliban comes with json encoders and decoders for circe, zio-json and play-json.
+Caliban comes with json encoders and decoders for circe, zio-json, jsoniter and play-json.
 If you use another json library, you will need to create encoders and decoders for it (which is very simple, you can simply look at the existing ones).
+
```

Comment:
Isn't an enclosing `:::` missing?

Reply:
I thought so too, but if I add one then this appears on the page 
![image](https://user-images.githubusercontent.com/67301607/206597168-3cc8e0dc-ac8a-462d-a8d1-ed184d536c6f.png)

Reply:
Interesting, I didn't know it supported nesting like that!

Reply:
Me neither, I only found out cause I accidentally pasted it within the enclosing marker and saw the browser just in time 😅